### PR TITLE
Cu-23dqzzp: feature mineluck API tests

### PR DESCRIPTION
**Pull request by rkamysz**
PR includes:

- API test suite for the `/mineluck` route
- mineluck fixtures
- minor fixes in api tests config

### CODE REVIEW

**Comment by dskvr** [comment]
Task linked: [CU-23dqzzp Write API tests for minelucks.ts api handler](https://app.clickup.com/t/23dqzzp)

File: src/api-handlers/mines.ts [hunk 1]
```diff
@@ -51,7 +52,7 @@ const getMines = async (fastify, request) => {
     } else {
       query.land_id = land_id;
     }
-    console.log(query);
+    // console.log(query);
```
Comment (montera82) [inline src/api-handlers/mines.ts:55]: nitpick, but may be time to start removing this kind of stuff.wdyt

**Review by montera82 — APPROVED** [review]
:shipit:

File: src/api-handlers/mines.ts (hunk 1 above)
Comment (ktamas77) [reply src/api-handlers/mines.ts:55]: agreed, pls remove commented out unused code / debug fragments

**Review by ktamas77 — APPROVED** [review]
great job, just a minor comment

**Comment by rkamysz** [comment]
you are right I will remove it

File: src/api-handlers/mines.ts (hunk 1 above)
Comment (rkamysz) [reply src/api-handlers/mines.ts]: removed

**Review by montera82 — APPROVED** [review]

**Pull request merged**